com.teamspeak.TeamSpeak.yaml

```
@@ -34,9 +34,9 @@ modules:
         only-arches:
           - i386
         filename: TeamSpeak3-Client.run
-        url: https://files.teamspeak-services.com/releases/client/3.3.2/TeamSpeak3-Client-linux_x86-3.3.2.run
-        sha256: c2fa7272a7542eb7f15170b386b8b2597e34aca8c5afcdff429b2c46672f5433
-        size: 94323084
+        url: https://teamspeak.com/en/downloads/
+        sha256: 53cbf5be268d37ae2357de75cffe6758c9a6f41ef47b31368e642f012fd8cc8d
+        size: 43063
         x-checker-data:
           type: rotating-url
```

I think you meant to use
```suggestion
          type: html
```